logic/networks.go

```diff
@@ -603,6 +603,10 @@ func networkNodesUpdateAction(networkName string, action string) error {
 		var node models.Node
 		err := json.Unmarshal([]byte(value), &node)
 		if err != nil {
+			if strings.Contains(err.Error(), "parsing time") {
```

Comment:
I don't think that gains you anything

Reply:
It would be a more accurate check, a util function to distinguish a legacy node on such an error would be even better. Then you would know if it's an error in current code base or an error because we've placed logic around legacy code, which hopefully we're not doing.